### ISSUE TITLE
Group names are not being picked up for code comments

### ISSUE THREAD

**Issue by al-fisher** [BHoM/CSharp_Toolkit#5]
Some of my scripts use the actual group name field as a comment. 
I also do use scribbles liberally as well - so good we support that already.  

Can we support both?

![image](https://user-images.githubusercontent.com/15924573/72210163-256c5800-34af-11ea-872a-d69d4125c1ae.png)

**Comment by adecler** [comment]
Yes, I can add that functionality. Should it be checking for scribbles first and then group name field (It can only use one of the two for creating the actual comment) ?

**Comment by al-fisher** [comment]
OK - thanks!
Sounds good
